js/count_monkeys.test.js

```diff
@@ -0,0 +1,13 @@
+const monkeyCount = require('./count_monkeys');
+
+
+describe('count_monkeys', () => {
+
+  test(' When it get 5 it should return array of numbers from 1 to 5', () => {
```

Comment:
The current name is miles better from the previous one 😄 However, it still could be better.

Consider the output from jest when it runs tests. The current test name will be:

> count_monkeys  When it get 5 it should return array of numbers from 1 to 5

There are a few problems:

1. There is a double space between `count_monkeys` and `When`
2. It does not read like a sentence
3. The grammar is incorrect 😄 it should be `gets`

For this one, I suggest:

```suggestion
  test('returns an array of numbers from 1 to 5 when given 5', () => {
```

Then, the full test name:

> count_monkeys returns an array of numbers from 1 to 5 when given 5

reads like a sentence. Moreover, it is a clear specification for what the function does.

Reply:
Try adjusting the second test's name as well 🙂